docs/api/ruby-api-hot-reload-view-helpers.md

```diff
@@ -13,8 +13,7 @@ static vs. hot is picked based on whether `ENV["REACT_ON_RAILS_ENV"] == "HOT"`
 
  <!-- These do not use turbolinks, so no data-turbolinks-track -->
  <!-- This is to load the hot assets. -->
- <%= env_javascript_include_tag(hot: ['http://localhost:3500/vendor-bundle.js',
-                                      'http://localhost:3500/app-bundle.js']) %>
+ <%= env_javascript_include_tag(hot: ['http://localhost:3500/webpack-bundle.js') %>
```

Comment:
@jbhatab no reason to change this doc file. the point was to show that it can take multiple files. It doesn't need to correspond to the generated file, and you missed the closing square bracket.